src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -404,13 +403,13 @@ public ForgeMod() {
         }
 
         MinecraftForge.EVENT_BUS.addListener(this::serverStopping);
-        ModLoadingContext.get().registerConfig(ModConfig.Type.CLIENT, ForgeConfig.clientSpec);
-        ModLoadingContext.get().registerConfig(ModConfig.Type.SERVER, ForgeConfig.serverSpec);
-        ModLoadingContext.get().registerConfig(ModConfig.Type.COMMON, ForgeConfig.commonSpec);
+        modEvent.registerConfig(ModConfig.Type.CLIENT, ForgeConfig.clientSpec);
+        modEvent.registerConfig(ModConfig.Type.SERVER, ForgeConfig.serverSpec);
+        modEvent.registerConfig(ModConfig.Type.COMMON, ForgeConfig.commonSpec);
         modEventBus.register(ForgeConfig.class);
         ForgeDeferredRegistriesSetup.setup(modEventBus);
         // Forge does not display problems when the remote is not matching.
-        ModLoadingContext.get().registerExtensionPoint(IExtensionPoint.DisplayTest.class, ()->new IExtensionPoint.DisplayTest(()->"ANY", (remote, isServer)-> true));
+        modEvent.registerExtensionPoint(IExtensionPoint.DisplayTest.class, ()->new IExtensionPoint.DisplayTest(()->"ANY", (remote, isServer)-> true));
```

Comment:
While you're at it, this can be simplified to:
```suggestion
        modEvent.registerDisplayTest(IExtensionPoint.DisplayTest.IGNORE_ALL_VERSION);
```